src/service/tempMailApi.js

```diff
@@ -6,11 +6,24 @@ class TemporalMailService {
   constructor() {
     this.BASE_API_URL = 'https://api.mail.tm';
     this.httpClient = require('axios');
+    this.emailDomain = '';
   }
+
+  async _getDomains() {
+    try {
+      const response = await this.httpClient.get(`${this.BASE_API_URL}/domains`);
+      const {domain} = response.data['hydra:member'];
+      this.emailDomain = domain;
```

Comment:
This is not following the style guides, of the others functions in this same file.